extensions/podman/src/extension.ts

```diff
@@ -572,8 +574,13 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
           const [machineName] = machines[0];
           console.log('Podman extension:', 'Autostarting machine', machineName);
           await execPromise(getPodmanCli(), ['machine', 'start', machineName], { logger });
+          machineAutoStarted = true;
+          machineAutoStartedName = machineName;
         }
       },
+      stop: async () => {
```

Comment:
When this is actually called? It looks to me that when you close podman desktop the deactivate method triggers the stopMachine, 🤔